docs/5-reference/1-contexts.md

```diff
@@ -39,14 +39,14 @@ The `ingresses` context contains information about the external exposed interfac
 | `ingresses`             | `object` | Information on the ingress rules of the component                                                                      |
 | `ingresses.<interface>` | `object` | Information on an ingress rule matching the interface. [See the interface values](#interface-values) for more details. |
 
-### `parameters` context
+### `secrets` context
 
-The `parameters` context contains all the values assigned to each parameter declared by your component.
+The `secrets` context contains all the values assigned to each secret declared by your component.
 
 | Property           | Type     | Description                                      |
 | ------------------ | -------- | ------------------------------------------------ |
-| `parameters`       | `object` | A dictionary containing the parameter values     |
-| `parameters.<key>` | `string` | Resolves to the value of the specified parameter |
+| `secrets`          | `object` | A dictionary containing the secret values        |
```

Comment:
I still see refs to parameters in `2-architect-yml.md`. Does that get regenerated still as part of semantic release?

Reply:
Looks like it should be generated with this file, though the file needs an update

https://github.com/architect-team/architect-cli/blob/rc/.github/workflows/generate-schema.yml

Reply:
Updated with this PR: https://github.com/architect-team/architect-cli/pull/559